src/components/Article.js

```diff
@@ -0,0 +1,14 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+
+const Article = (props) (
+    <>
+        <h1>{props.title || null}</h1>
+        <p> {props.text.length < 120 ? props.text : null}</p>
+        <span>{props.date || null}</span>
+    </>

```

Comment:
```suggestion
    <>
      <h1>{props.title || null}</h1>
      <p> {props.text.length < 120 ? props.text : null}</p>
      <span>{props.date || null}</span>
    </>
```
Два отступа достаточно